go/greeter/.choreo/endpoints.yaml

```
@@ -0,0 +1,37 @@
+# Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+version: 0.1
+
+# +required List of endpoints to create
+endpoints:
+  # +required Unique name for the endpoint. (This name will be used when generating the managed API)
+- name: GoGreeterEndpoint
+  # +required Numeric port value that gets exposed via this endpoint
+  port: 9090
+  # +required Type of the traffic this endpoint is accepting. Example: REST, GraphQL, etc.
+  # Allowed values: REST, GraphQL, GRPC
+  type: REST
+  # +optional Network level visibility of this endpoint. Defaults to Project
+  # Accepted values: Project|Organization|Public.
+  visibility: Public
+  # +optional Context (base path) of the API that is exposed via this endpoint.
+  # This is mandatory if the endpoint type is set to REST or GraphQL.
+  context: /greeter
```

We need to change this to root context as the OAS already has the `/greeter` in the path. Otherwise, we need to update the OAS to only contain the `/greet`
```suggestion
  context: /
```